### ISSUE TITLE
request for function

### ISSUE THREAD

**Issue by HIN0209** [knjcode/mxnet-finetuner#41]
Hello,

I really enjoy this! I hope that adding the following features would be more helpful.
1: The name of the training library other than /image can be selected by config.yml
2: plotting training/validation losses
3: The model folder is full of  saved checkpoints for each epoch (already  400 GB in my case, too painful with someone using a compact PC setting). config.yml to specify how to keep the checkpoint data (save all, save the best one only, save the last one only, etc.).

Thanks!!

**Comment by knjcode** [comment]
Thanks for suggesting features. And sorry for my late reply.

Training/validation loss plotting function is currently being implemented.

I plan to implement the functions of No. 1 and 3 as well, but I do not know when that will be.
